client/src/modules/StudentDashboard/components/MentorCard.tsx

```diff
@@ -60,7 +60,7 @@ export function MentorCard(props: Props) {
   return (
     <CommonCard
       title="Mentor"
-      icon={<TeamOutlined />}
+      icon={null}
```

Comment:
could we just omit the prop without specifying null?

Reply:
yes, I'll set icon prop type to be optional on component CommonCard and omit it on those cards. Seams like CommonCard is used only on this cards, so it wouldn't broke anything

Reply:
maybe I should remove that icon prop at all since it's not going to be used anywhere